src/sass/app/components/peek_box.scss

```diff
@@ -203,10 +202,10 @@ $COLOR_PEEK_LIGHT_GRAY: #c2c2c2;
         align-items: center;
         justify-content: center;
         font-weight: bold;
-        border-right: 1px solid $COLOR_GRAY;
-        border-top: 1px solid $COLOR_GRAY;
-        border-bottom: 1px solid $COLOR_GRAY;
-        color: $COLOR_PEEK_LIGHT_GRAY;
+        border-right: 1px solid $color-grey-5;
+        border-top: 1px solid $color-grey-5;
+        border-bottom: 1px solid $color-grey-5;
+        color: $color-grey-5;
```

Comment:
can you use them this way? or you need to use the var version from themes?

Reply:
You're right, I should use CSS vars here. :thu